AmericanMedicalAssociation/ama-style-guide-2
# [EWL-4614] SG2 | Create Page Content Tab(s) Organism(s)

## Ticket(s)

**Jira Ticket**
- [EWL-4614: SG2 | Create Page Content Tab(s) Organism(s)](https://issues.ama-assn.org/browse/EWL-4614)

## Description
As a user, I want to find out information about an event by clicking on the different page tabs.

This update creates the twig template, descriptive markdown, and relevant scss required to make the page content tabs organism and  along with the backstop test case and reference images. This organism has been added to the event page and the event page documentation has been updated.

Came across bug for info cards as speaker when used on event pages. This bug has been documented in ticket [EWL5113 SG2 | Correct info card as speaker alignment](https://issues.ama-assn.org/browse/EWL-5113). This bug causes the content in speakers tab to look misaligned.

Mobile view accordion functionality leverages the jQuery UI accordion library.

## To Test

- Pull `feature/EWL-4614-create-page-content-tabs-organism` branch
- Run `gulp serve` in ama-style-guide-2/styleguide to ensure a local version of style guide is running.
- Navigate to [Organism > Page Content Tabs](http://localhost:3000/?p=page-content-tabs-organisms) or [Page > Event](http://localhost:3000/?p=event-pages).
- Confirm organism and page looks as shown in Zeplin mock up.
- Click on the gear icon in the top right corner of the screen and then click "show pattern info". Relevant information about the event and page content tabs should appear.

## Visual Regressions

Updated reference images for the event page backstop tests have been included in this PR.

Multiple tests failed when running backstop including some issues with chromy waitTimeOut errors. Due to the errors experienced, non-related test failures may not have been noted in a bug ticket.

## Relevant Screenshots/GIFs
### Updated Event page
![image](https://user-images.githubusercontent.com/4438120/40248736-7c4c958a-5a96-11e8-99ad-c6814de6813a.png)

### Page content tabs organism
![image](https://user-images.githubusercontent.com/4438120/40251654-886ffa4c-5a9f-11e8-8309-4aa99626933c.png)




## Remaining Tasks
- [ ] Update glossary to include page content tabs definition.


## Additional Notes
N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
